<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Lukáš Tinkl &lt;ltinkl@redhat.com&gt;</author>
 <class>VpncWidget</class>
 <widget class="QWidget" name="VpncWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>313</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Gateway:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>gateway</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="gateway">
     <property name="toolTip">
      <string>IP/hostname of IPsec gateway.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>User name:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>user</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="user">
     <property name="toolTip">
      <string>User name for the connection.</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>User password:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>userPassword</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="PasswordField" name="userPassword">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>User password for the connection.</string>
       </property>
       <property name="passwordModeEnabled" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Group name:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>group</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="group">
     <property name="toolTip">
      <string>Group name</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Group password:</string>
     </property>
     <property name="buddy">
      <cstring>groupPassword</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="PasswordField" name="groupPassword">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Group password.</string>
       </property>
       <property name="passwordModeEnabled" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QCheckBox" name="useHybridAuth">
     <property name="toolTip">
      <string>Enable hybrid authentication, i.e. use certificate in addition to password.</string>
     </property>
     <property name="text">
      <string>Use hybrid authentication</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_6">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>CA file:</string>
     </property>
     <property name="buddy">
      <cstring>caFile</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="KUrlRequester" name="caFile">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>CA certificate in PEM format.</string>
     </property>
     <property name="filter">
      <string>*.pem *.crt *.cer</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnAdvanced">
       <property name="text">
        <string>Advanced...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="9" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>PasswordField</class>
   <extends>QLineEdit</extends>
   <header>passwordfield.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>gateway</tabstop>
  <tabstop>user</tabstop>
  <tabstop>userPassword</tabstop>
  <tabstop>group</tabstop>
  <tabstop>groupPassword</tabstop>
  <tabstop>useHybridAuth</tabstop>
  <tabstop>caFile</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>useHybridAuth</sender>
   <signal>toggled(bool)</signal>
   <receiver>caFile</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>391</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>421</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useHybridAuth</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_6</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>33</x>
     <y>424</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
